packages/web-components/src/components/ic-popover-menu/ic-popover-menu.tsx

```diff
@@ -12,6 +12,14 @@ import {
 import { getSlotElements } from "../../utils/helpers";
 import { createPopper } from "@popperjs/core";
 
+enum IcPopoverMenuClickEnum {
+  Left = 0,
+  Middle = 1,
+  Right = 2,
+  Back = 3,
+  Forward = 4
+}
+
```

Comment:
I think this should be exported from an ic-popover-menu.types.ts file like we've done for other components (see ic-button for an example)